packages/plasma-new-hope/src/components/Accordion/ui/AccordionItem/AccordionItem.types.ts

```diff
@@ -36,7 +36,7 @@ type Props = {
     /**
      * Заголовок accordion
```

Comment:
Заметил, что где в типах используется термин «элемент accordion», а где-то просто «accordion». 
Может для `AccordionItem` ради единообразия везде использовать «элемент accordion»? 

```suggestion
     * Заголовок элемента accordion
```

И аналогично для других пропсов